### DISCUSSION TITLE
[공지사항] ✨깜짝 문제✨ 두둥등장

### DISCUSSION THREAD

**Discussion by dev-LeoKang** [4Moyede/HexatonClass-01#103] [00_공지사항]
안녕하세요 강형구 멘토입니다.
수업에 부족한 것은 없는지, 멘티들의 실력 향상에 더 도움이 될 수는 없는지 유혁준 멘토와 함께 고민을 나누다가, 
✨깜짝 문제✨로 여러분을 도와드리기로 결정했습니다ㅎㅎ
**지금까지 더 노력하지 않고 안일했던 태도를 반성하며, 앞으로 멘티들을 위해 분발하는 멘토들이 되겠습니다.**

문제는 간단합니다. 

**지금까지 배운 모든 것을 사용하여서 계산기 Class를 만드시면 됩니다. (변수, 메소드, if, while)**

실행 화면 예시:

input : 3
input : +
input : 5
input : =
[result] 8
input : *
input : 5
input : =
[result] 40
input : 5
input : -
input : 2
input : =
[result] 3

실행화면은 하나의 예시일 뿐이며, 원하시는 방향으로 마음껏 만드셔도 됩니다.
지키셔야할 조건은 **단 하나** 입니다.

* **실제 계산기를 사용하는 방식과 같도록 할 것.**

이 문제에 대한 제 코드는 모든 멘티분들이 제출 하셨을 때 공개하겠습니다.ㅎㅎ
화이팅🔥

**Comment by dev-LeoKang** [comment]
@lsj0822 @NA-DK @dlrudgnsdlek @jiwoo4424 @JaebeomHan

**Comment by lsj0822** [comment]
✨확인했습니다✨

**Comment by ghost** [comment]
확인!

**Comment by NA-DK** [comment]
확인했습니다!

**Comment by jiwoo4424** [comment]
확인했습니다 !
